cmd/main.go

```diff
@@ -37,3 +38,10 @@ func main() {
 		log.Fatal().Err(err).Msg("server failed")
 	}
 }
+
+func OutputSwitcher() io.Writer {
+	if _, err := os.Stat("/.dockerenv"); err != nil {
```

Comment:
It's a valid strategy, but let's use something easier. We already have `ENV GIN_MODE=release` in Dockerfile, you should reuse it. In the future we will want to use docker-compose, but still pretty print to console.

Reply:
And please rename the function to `selectLogOutput`